### ISSUE TITLE
[completed-docs] exported variable parsed as internal variable

### ISSUE THREAD

**Issue by bumbleblym** [palantir/tslint#2335]
### Bug Report

- __TSLint version__: 4.5.1
- __TypeScript version__: 2.2.1
- __Running TSLint via__: (pick one) CLI / ~~Node.js API / VSCode / grunt-tslint / Atom / Visual Studio / etc~~

#### TypeScript code being linted

```ts
export const Null = null
```

with `tslint.json` configuration:

```json
{
  "rules": {
    "completed-docs": [true, {
      "variables": {
        "visibilities": [
          "exported"
        ]
      }
    }]
  }
}
```

#### Actual behavior
`"all"`: lint error
`"exported"`: no lint error
`"internal"`: lint error
#### Expected behavior
`"all"`: lint error
`"exported"`: lint error
`"internal"`: no lint error

`exported` variable is being parsed as `internal`. [repro](https://github.com/bumbleblym/tslint-bug)

Also,  [documented `visibilities` type is inconsistent](https://palantir.github.io/tslint/rules/completed-docs/):
[schema](https://github.com/palantir/tslint/blob/ee721bb4fff7a3b75901cc0b29d809c157eefd5d/src/rules/completedDocsRule.ts#L104): `enum`
[example](https://github.com/palantir/tslint/blob/ee721bb4fff7a3b75901cc0b29d809c157eefd5d/src/rules/completedDocsRule.ts#L205): `array enum`
[implementation](https://github.com/palantir/tslint/blob/ee721bb4fff7a3b75901cc0b29d809c157eefd5d/src/rules/completedDocsRule.ts#L277): `array enum`

perhaps #629 should consider docs as well.

**Comment by JoshuaKGoldberg** [comment]
I'll take a look after #2415 goes in.

**Comment by reduckted** [comment]
This bug is caused by the fact that the `VariableDeclaration` does not have modifiers. The modifiers are on the `VariableStatement` which is a grandparent of a `VariableDeclaration`.

    VariableStatement               <-- The relevant modifiers are here.
     └─ VariableDeclarationList
         └─ VariableDeclaration     <-- This node is checked.

I've made a fix for this, but it requires some changes that I've made in PR #2911 to allow the modifiers from a different node to be used. Once that PR is merged, I'll submit another PR for this bug.

**Comment by JoshuaKGoldberg** [comment]
Bump @reduckted - still interested in a followup PR?

**Comment by reduckted** [comment]
Doh! Totally forgot about this. Yes, I'll take a look into it again. :)

**Comment by reduckted** [comment]
Hmm, looks like I already fixed this in #2950. ¯\\\_(ツ)\_/¯

**Comment by ajafff** [comment]
Looking at the tests added in that PR I can confirm that this is fixed.
